app/javascript/article-form/components/__tests__/Form.test.jsx

```diff
@@ -6,13 +6,29 @@ import { Form } from '../Form';
 let bodyMarkdown;
 let mainImage;
 
+// Axe flags an error for the multi-line combobox we use for Autosuggest, since a combobox should be a single line input.
```

Comment:
Question (non-blocking): I'm wondering if we even need to keep this test. We have some E2E tests even prior to your changes for the editor. Maybe for a subsequent PR.

Reply:
That's a good point... The axe ones are probably good to keep, to make sure we've no violations other than the known combobox issues, but there's a lot in this directory that maybe we don't need, or should be covered in E2E tests instead (e.g. Preview, Help